cmd/dvotecli/commands/process.go

```diff
@@ -0,0 +1,142 @@
+package commands
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"go.vocdoni.io/dvote/client"
+	"go.vocdoni.io/dvote/types"
+	"go.vocdoni.io/dvote/util"
+	"nhooyr.io/websocket"
+)
+
+var processCmd = &cobra.Command{
+	Use:   "process list|keys|results|finalresults|liveresults",
+	Short: "process subcommands",
+}
+
+var processListCmd = &cobra.Command{
+	Use:   "list [entityId]",
+	Short: "list processes of entity",
+	RunE:  processList,
+}
+
+var processKeysCmd = &cobra.Command{
+	Use:   "keys [processId]",
+	Short: "list keys of processes",
+	RunE:  processKeys,
+}
+
+var processResultsCmd = &cobra.Command{
+	Use:   "results [processId]",
+	Short: "get the results of a process",
+	RunE:  getResults,
+}
+
+func init() {
+	rootCmd.AddCommand(processCmd)
+	processCmd.AddCommand(processListCmd)
+	processCmd.AddCommand(processKeysCmd)
+	processCmd.AddCommand(processResultsCmd)
+}
+
```

Comment:
Add "entityList" also to fetch the list of entities registered in the blockchain: https://docs.vocdoni.io/#/architecture/services/gateway?id=get-entity-list

Reply:
I will use `entityList` instead of `getScrutinizerEntities`  in `entity.go`, also with pagination